src/pages/deployment/builds.md

```diff
@@ -94,11 +94,10 @@ the deployment starts.
 A Procfile is in the format of
 
 ```
-process1: command
-process2: command
+process: command
```

Comment:
very minor nit: Use `web` here since we should push users in that direction since it works for web servers

Reply:
Unsure if we should do that as it's explaining a procfile, also, the web Procfile sample is directly after